services/efs-guardian/src/simcore_service_efs_guardian/core/settings.py

```diff
@@ -69,6 +71,7 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
     EFS_GUARDIAN_AWS_EFS_SETTINGS: AwsEfsSettings | None = Field(
         auto_default_from_env=True
     )
+    EFS_GUARDIAN_RABBITMQ: RabbitSettings | None = Field(auto_default_from_env=True)
```

Comment:
question: does it make sense to make this optional? e.g. can it work without RabbitMQ? actually same question for the EFS settings above? they should not be optional no?